week-1/interpret/to-pounds.js

```diff
@@ -1,21 +1,21 @@
-const penceString = "399p";
+const penceString = "399p"; //initializes a string variable with the value "399p"
 
 const penceStringWithoutTrailingP = penceString.substring(
   0,
   penceString.length - 1
-);
+);    // Initializes a new variable to remove the last character from the string
 
-const paddedPenceNumberString = penceStringWithoutTrailingP.padStart(3, "0");
+const paddedPenceNumberString = penceStringWithoutTrailingP.padStart(3, "0"); // Initializes a new variable to pad the whole number
```

Comment:
Why does the code need to do this?

Reply:
So that it can seperate the pence from pounds